Sources/BraintreeThreeDSecure/V2UICustomization/BTThreeDSecureV2UICustomization_Internal.h

```diff
@@ -4,8 +4,10 @@
 #import <BraintreeThreeDSecure/BTThreeDSecureV2UICustomization.h>
 #endif
 
+#import <CardinalMobile/CardinalMobile.h>
+
 @interface BTThreeDSecureV2UICustomization ()
 
-@property (nonatomic, strong) id cardinalValue;
+@property (nonatomic, strong) UiCustomization *cardinalValue;
```

Comment:
random question: We're keeping the BT customization wrapper types?

Reply:
Yeah, I think they're nice to have. It lets us have a little more control over the public API. Plus, removing them now would be a breaking change.

Reply:
Yeah that makes sense. There are a few areas in the Android SDK where wrapper types would make sense to keep 3rd-party deps from introducing breaking changes.